shared_workflow/install_shared.py

```diff
@@ -77,6 +79,12 @@ def install_simulation(
     root_params_dict[RootParams.stat_vs_est.value] = vs30_file_path
     root_params_dict[RootParams.stat_vs_ref.value] = vs30ref_file_path
     root_params_dict["hf"][RootParams.seed.value] = seed
+    if components != None:
+        if not set(components).issubset(set(Components.iterate_str_values())):
+            message = f"{components} are not all in {Components}"
```

Comment:
This gets checked as part of IM_calc argparse. Is it worth having a double check here? Especially because parameters can change between install and run.

Reply:
this check will raise an error at 'install' stage, instead of breaking when it reaches IM_calc step.
for a fully automated usage i think this is needed. and if the user decided to change parameters inbetween thats their own responsibility